### ISSUE TITLE
"Report issue" for GH Codespaces opens issue reporter on 1st time clicking

### ISSUE THREAD

**Issue by ulugbekna** [microsoft/vscode#180796]
Testing #180561

> The Report Issue node for GitHub codespaces now goes to GitHub Discussions rather than opening the issue reporter

The first time I click on "Report issue" and pick GH Codespaces from the quickpick, I get the issue reporter. After the first time, it does open the browser. I was thinking I just need to wait for extensions to get initialized, but this happens even after the TS Server fully starts working. 

https://user-images.githubusercontent.com/16353531/234232006-11b89444-32d6-4501-b876-49edb198c750.mov

## Version info

Version: 1.78.0-insider (Universal)
Commit: 2902f278ad8e2d830fa6883d3d8fd7b5c2ca59e9
Date: 2023-04-25T05:41:22.331Z
Electron: 22.4.8
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.4.0
Sandboxed: Yes

**Comment by gregvanl** [comment]
I am able to repro this sometimes just after doing Reload Window. Only happens the first time going to Report Issue and selecting GitHub Codespaces

**Comment by joyceerhl** [comment]
Yeah, once @alexr00 adds `onView` for builtin activation events and Codespaces adopts it, this problem will go away. The Codespaces extension takes longer than our timeout (500ms) for safeguarding against remote extensions which never return a URL to display in this picker.
